web/client/localConfig.json

```
@@ -342,7 +342,17 @@
                         "alwaysVisible": true
                     }
                 }
-            }, "Home", "FeatureEditor",
+            },
+            "Home",
+            {
+              "name": "FeatureEditor",
+              "cfg": {
+                "editingAllowedRoles": [
+                  "ADMIN",
+                  "USER"
+                ]
+              }
+            },
```

```suggestion
            }, "Home", "FeatureEditor",
```
Please do not commit these changes.